### ISSUE TITLE
Query Loop: Problem when query loop requests posts from different post types (and other stuff)

### ISSUE THREAD

**Issue by quantumleap33** [tomusborne/generateblocks#352]
Something is going on with the query post type selector. Steps to reproduce:
1. Create a test page and insert a query loop. Posts retrieved to be normal posts.
2. Save the page.
3. Create a custom post type with the Custom Post Type UI plugin, eg: Events
4. Add some content, maybe a couple of event posts with featured images, some lorem ipsum text (I've also added a couple of ACF datetime custom fields)
5. Go back to the page where the post query loop was and try adjusting the loop to retrieve events instead of posts. 
6. Only normal posts are displayed in the preview. Even switching to any other post type doesn't have any effect. It's like the query is stuck to only requesting normal posts. In fact, only one post is previewed instead of all the posts. Something strange is going on.
7. Also at this point the editor decided to go all psychedelic on me as shown in this video. It happens even after I refresh the page (prepare yourselves, lol):
https://www.loom.com/share/dc515ba52e184c4fb4e08f7ba9721115
9. Create a new page and insert a query loop block.
10. Try setting the post source to events or any other post type
11. This time you can see the loop populating with the custom post type data.

**Comment by quantumleap33** [comment]
Maybe the whole thing is the result of having created the first post loop with a previous version of the query loop PR. Updating to the latest one and starting again doesn't cause that.

**Comment by JeanPaiva** [comment]
@quantumleap33 in a fresh loop none of this happens? Tried on my local and could not reproduce.

**Comment by quantumleap33** [comment]
Yeah, probably had something to do with the fact that the previous loop was made with a previous PR. Fresh loop works fine. Maybe we can ignore this for now!

**Comment by JeanPaiva** [comment]
Ok, thank you. For this phase until beta, we will be doing breaking changes, so its always nice to start fresh :)

**Comment by quantumleap33** [comment]
Yeah, sure, thanks!
